src/components/Checkbox.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import colors from '../styles/colors';
+
+interface CheckboxProps {
+	id: string;
+	disabled?: boolean;
+	label?: string;
+	name?: string;
+	checked?: boolean;
+	onChange: (checked: boolean) => unknown;
+}
+
+export default function Checkbox({
+	id,
+	disabled = false,
+	label,
+	name,
+	checked,
+	onChange,
```

Comment:
can you add `...rest` and add that to the`StyledContainer`, so if the consumer component plugs that into styled component again cause he needs to fine tune this component, the styles are getting applied